src/pages/index.js

```diff
@@ -279,9 +155,49 @@ export default function IndexPage({ pkgVersion }) {
               }
             }}
           />
+          <div className="pb-6">
+            <SearchField
+              className="pt-6 w-full sm:px-6 md:px-0 md:mb-0"
+              inputClassName="w-full"
+              value={searchText}
+              onChange={(e) => {
+                const search = e.target.value;
+
+                setSearchText(e.target.value);
+
+                const filteredIcons =
+                  selectedTab !== 'all'
+                    ? getChromicons()?.filter((icon) =>
+                        icon?.categories?.includes(selectedTab)
+                      )
+                    : getChromicons();
+
+                if (!search) {
+                  setVisibleIcons(filteredIcons);
+                  return;
+                }
+
+                setVisibleIcons(
+                  filteredIcons?.filter(
+                    (icon) =>
+                      icon.rawName
+                        ?.toLowerCase()
+                        .includes(search.toLowerCase()) ||
+                      icon.name?.toLowerCase().includes(search.toLowerCase()) ||
+                      icon?.keywords
+                        ?.toLowerCase()
+                        ?.includes(search.toLowerCase())
+                  )
+                );
+              }}
+            />
+          </div>
         </div>
         {visibleIcons?.length > 0 ? (
-          <div className="flex flex-wrap justify-center px-4 my-4 max-w-6xl mx-auto sm:px-6 lg:px-16">
+          <div
+            className="flex flex-wrap justify-center mx-auto bg-white"
+            style={{ zIndex: '9999' }}
```

Comment:
Same here - probably shouldn't be this reliant on that high of a z-index

Reply:
Dropped the z-index. It is needed for the image in the hero. Didn't need to be so high of a z-index.